Sources/UI/SwiftUI/View Modifiers/PinchToZoomModifier.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  PinchToZoomModifier.swift
+//  PovioKit
+//
+//  Created by Yll Fejziu on 13/05/2024.
+//  Copyright © 2024 Povio Inc. All rights reserved.
+//
+
+import SwiftUI
+
+/// https://stackoverflow.com/a/59878898
+@available(iOS 14.0, *)
+public struct PinchToZoomModifier: ViewModifier {
+  @StateObject private var viewModel = PinchZoomViewModel()
+
+  public func body(content: Content) -> some View {
+    content
+      .scaleEffect(viewModel.scale, anchor: viewModel.anchor)
+      .offset(viewModel.offset)
+      .animation(viewModel.isPinching ? .none : .spring(), value: viewModel.scale)
+      .overlay(
+        PinchZoom(viewModel: viewModel)
+      )
+  }
+}
+
+@available(iOS 14.0, *)
```

Comment:
We don't really want to support SwiftUI on iOS 14. Only 15+. Please update.

Reply:
Updated